packages/material-ui/src/ButtonBase/ButtonBase.js

```diff
@@ -43,6 +43,9 @@ export const styles = {
       pointerEvents: 'none', // Disable link interactions
       cursor: 'default',
     },
+    '&:focus-visible': {
```

Comment:
This works different than our polyfill. At least if you tab back into the window our polyfill won't apply whereas the chrome implementation does.

Reply:
I think that we should follow the Chrome behavior. I can reproduce it too with the class name on https://wicg.github.io/focus-visible/demo/.